### ISSUE TITLE
Incorrect phenology for Snowdon Lily

### ISSUE THREAD

**Issue by burkmarr** [BiologicalRecordsCentre/bsbi-atlas-drupal8#23]
Kevin: Just noticed something strange on website. The phenology plot for Snowdon Lily has flowering 6-12 but it’s correct in the book (June only). Could we add this to the list of website edits?

Tom: That's odd, as the flowering time is set correctly in the DDb (6 - 6).

**Comment by sacrevert** [comment]
@burkmarr jf you find some general bug behind this, could you re-download the phenology information export for me please? (The one that will go on the EIDC). No great rush as the EIDC are taking an age currently

**Comment by sacrevert** [comment]
There are the values in the caption export you provided, so perhaps this bug is something to do with not specifying the end month explicitly?

phenFlowerStart | phenFlowerEnd
6 |

**Comment by burkmarr** [comment]
Yes that's likely it Oli. I hope to look at this today.

**Comment by sacrevert** [comment]
If it is, don't worry about redownloading the data. I think it's clear enough within the the current dataset. I'll just point out in the metadata than single month ranges are just given in phenFlowerStart only

**Comment by burkmarr** [comment]
This is fixed now. The problem, as you guessed Oli, is that the end flowering date was blank in the data extract from the DDB and the code was not expecting this. I updated the code to check for this and if the end month is blank, set it to be the same as the start month (leafing and flowering).

When I looked back at the DDB extract we used in August last year (when the book was produced) - this did not have a blank end month - it was set to the same value as the start month which is why the problem did not appear with the charts in the book.
